### ISSUE TITLE
Volume Health Monitoring support

### ISSUE THREAD

**Issue by alexminder** [piraeusdatastore/linstor-csi#190]
Add support for new Kubernetes feature [Volume Health Monitoring](https://kubernetes.io/docs/concepts/storage/volume-health-monitoring/)

**Comment by WanzenBug** [comment]
We already have limited support for it: https://github.com/piraeusdatastore/linstor-csi/blob/a2252713ab8a006bd7cb1e3cad0072b66a39d301/pkg/client/linstor.go#L1716

Per-node volume condition reporting is not implemented, as it is not trivial to implement without involving the LINSTOR API, and ideally the CSI node part does not interact with the API directly.

**Comment by alexminder** [comment]
hello, any updates for issue?

**Comment by WanzenBug** [comment]
No. We support volume health monitoring in general, but on a per node basis. Monitoring is better achieved by things like Prometheus, for which we do have integration in the form of DRBD reactor.
